thresholds:
  probability: 0.7
  matching: 0.7
  max_iou_distance: 0.7
  max_age: 20
  n_init: 5
queue:
  size: 5